scripts/db_migrator.py

```diff
@@ -160,8 +160,10 @@ def migrate_feature_table(self):
         '''
         feature_table = self.configDB.get_table('FEATURE')
         for feature, config in feature_table.items():
-            state = config.pop('status', 'disabled')
-            config['state'] = state
+            state = config.get('status')
+            if state is not None:
+                config['state'] = state
+                config.pop('status')
```

Comment:
This change has different behavior?

Reply:
Aligned with https://github.com/Azure/sonic-utilities/pull/1674